app/models/brand.rb

```diff
@@ -1,3 +1,16 @@
 class Brand < ApplicationRecord
   has_many :saved_brands
+
+  validates :name, presence: true, uniqueness: true
+  validates :website_url, presence: true
+  validates :price_category, presence: true, numericality: { only_integer: true }, inclusion: { in: 1..5 }
+  validates :rating_earth, presence: true, numericality: { only_integer: true }, inclusion: { in: 1..5 }
+  validates :rating_people, presence: true, numericality: { only_integer: true }, inclusion: { in: 1..5 }
+  validates :rating_animals, presence: true, numericality: { only_integer: true }, inclusion: { in: 1..5 }
+  validates :rating_materials, presence: true, numericality: { only_integer: true }, inclusion: { in: 1..5 }
+  validates :about, presence: true, length: { minimum: 100 }
+  validates :address, presence: true
+  validates :style, inclusion: { in: %w[modern outdoor minimalist retro boujie arty scandinavian grunge formal
```

Comment:
name, website, style, approves to be the only ones with presence: true (as per our phone call) and remove about minimum length @emilyunderhill

Reply:
for style does the word inclusion automatically make the presence: true? if not can we make presence: true for style pleease @emilyunderhill